tensorflow/Dockerfiles/cpu-devel-ppc64le-jupyter.Dockerfile

```diff
@@ -111,7 +111,6 @@ RUN mkdir /.local && chmod a+rwx /.local
 RUN apt-get update && apt-get install -y --no-install-recommends wget
 WORKDIR /tf/tensorflow-tutorials
 RUN wget https://raw.githubusercontent.com/tensorflow/docs/master/site/en/tutorials/keras/classification.ipynb
-RUN wget https://raw.githubusercontent.com/tensorflow/models/master/samples/core/tutorials/keras/basic_text_classification.ipynb
```

Comment:
I guess this removal is to match upstream?

Reply:
It was removed over the weekend with: https://github.com/tensorflow/models/pull/8234/files, the dockerfiles in tensorflow/tensorflow didn't have this line. (So I'm unsure why I didn't remove it earlier.)